### PR TITLE
Remove query-stream and cursor packages for now

### PR DESCRIPTION
**Pull request by alexr-bq**
These packages are out of scope for MVP development of this driver, and should be removed for now as they do not work completely. This commit can later be reverted when these packages are ready to be fixed.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -941,7 +941,7 @@
   dependencies:
     "@types/node" ">= 8"
 
-"@types/chai@^4.2.13", "@types/chai@^4.2.7":
+"@types/chai@^4.2.7":
```
Comment (jonathanl-bq) [inline yarn.lock:944]: why was this changed?

Comment (alexr-bq) [reply yarn.lock:944]: All these changes were automatic as part as re-running the build after removing the packages, I'm assuming because of dependency requirements being removed.

Comment (DMickens) [reply yarn.lock:944]: That's correct, and as such it doesn't really matter whether or not this is part of the PR as it'll happen as soon as anyone builds after pulling master with removed packages

**Review by DMickens — APPROVED** [review]
I'm glad to have this done. Tightens up the MVP scope and leaves us with a more complete package. These changes look good to me
